js/common.js

```diff
@@ -423,6 +423,7 @@ var App = new function() {
 
     this.onLogin = function() {
         document.body.classList.add('loggedin');
+        document.getElementById("theme-color").setAttribute('content','#82C23A');
```

Comment:
nit: you used lower case letters in the <meta> tag, so do the same here (or the other way around, but be consistent)